From 6bdbcedf0d5aa79c39a7c5e8291e9aca70220342 Mon Sep 17 00:00:00 2001
From: Shizhao Liu <lshizhao@amazon.com>
Date: Fri, 23 Jan 2026 18:56:26 -0800
Subject: [PATCH 17/17] set failCGroupv1 to false

k8s 1.35 will disable support for cgroup, but since we will
still allow building images that uses cgroup v1, we set this
flag to false so kubelet does not fail when the node uses
cgroup v1.

Signed-off-by: Shizhao Liu <lshizhao@amazon.com>
---
 images/capi/ansible/roles/kubernetes/defaults/main.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/images/capi/ansible/roles/kubernetes/defaults/main.yml b/images/capi/ansible/roles/kubernetes/defaults/main.yml
index 6cf40cf14..81c80d60a 100644
--- a/images/capi/ansible/roles/kubernetes/defaults/main.yml
+++ b/images/capi/ansible/roles/kubernetes/defaults/main.yml
@@ -43,6 +43,6 @@ kubernetes_cni_http_checksum: sha256:{{ kubernetes_cni_http_source }}/{{ kuberne
 
 kubeadm_template: etc/kubeadm.yml
 
-kubelet_extra_args: ""
+kubelet_extra_args: "--fail-cgroupv1=false"
 
 kubernetes_enable_automatic_resource_sizing: false
-- 
2.45.2

